Automatic update of fbcode/onnx to 65020daafa9183c769938b4512ce543fd5740f8f

houseroad opened the pull request:
Summary:
Previous import was 57ebc587fcf3913b4be93653b0dd58c686447298

Included changes:
- **[65020daa](https://github.com/onnx/onnx/commit/65020daa)**: better error message for undefined inputs (#2540) <Yuxin Wu>
- **[8afff0e9](https://github.com/onnx/onnx/commit/8afff0e9)**: bump ORT version (#2538) <Lu Fang>
- **[3d9ca57e](https://github.com/onnx/onnx/commit/3d9ca57e)**: fix name of directory (#2537) <Prasanth Pulavarthi>
- **[df8fa2c9](https://github.com/onnx/onnx/commit/df8fa2c9)**: Repository guidelines (#2539) <Prasanth Pulavarthi>
- **[49cc2f02](https://github.com/onnx/onnx/commit/49cc2f02)**: Update CircleCI job to use Python3.6 (#2527) <bddppq>
- **[25ff79a4](https://github.com/onnx/onnx/commit/25ff79a4)**: Fix wrong model version, it's not 12 (the onnx_opset_version()), not 11 (the opset version of the latest stable), but 10 (#2478) <daquexian>
- **[7cebaed5](https://github.com/onnx/onnx/commit/7cebaed5)**: Fix Windows py3.5 CI (#2529) <bddppq>
- **[eddae00e](https://github.com/onnx/onnx/commit/eddae00e)**: Correct the order of arguments of InferShapes (#2500) <Shinichiro Hamaji>
- **[41b5afe6](https://github.com/onnx/onnx/commit/41b5afe6)**: Include <ostream> in common/status.h (#2519) <Casey Carter>
- **[423f1977](https://github.com/onnx/onnx/commit/423f1977)**: add 8 bit support to maxpool op (#2510) <Ashwini Khade>
- **[78593c2f](https://github.com/onnx/onnx/commit/78593c2f)**: add 8 bit support to reducemin and reducemax ops (#2516) <Ashwini Khade>

Test Plan: cont build

Differential Revision: D19380034

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 327a1128:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=327a1128e0cb2b757a4c2e8dd058fa46f4e1d47c).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4224783) caffe2_onnx_py3_6_clang7_ubuntu16_04_test (1/1)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=84433185) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=84433185))
<details>
<summary>
<code>Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_uint8_cpu FAILED [ 45%]</code>
</summary>

```
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_precomputed_same_upper_cpu SKIPPED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_precomputed_same_upper_cuda SKIPPED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_precomputed_strides_cpu PASSED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_precomputed_strides_cuda SKIPPED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_same_lower_cpu SKIPPED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_same_lower_cuda SKIPPED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_same_upper_cpu SKIPPED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_same_upper_cuda SKIPPED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_strides_cpu PASSED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_strides_cuda SKIPPED [ 45%] 
Jan 13 21:58:27 ../.local/lib/python3.6/site-packages/caffe2/python/onnx/tests/onnx_backend_test.py::OnnxBackendNodeModelTest::test_maxpool_2d_uint8_cpu FAILED [ 45%] 
Jan 13 21:58:27  
Jan 13 21:58:27 =================================== FAILURES =================================== 
Jan 13 21:58:27 ______________ OnnxBackendNodeModelTest.test_maxpool_2d_uint8_cpu ______________ 
Jan 13 21:58:27  
Jan 13 21:58:27 test_self = <caffe2.python.onnx.tests.onnx_backend_test.OnnxBackendNodeModelTest testMethod=test_maxpool_2d_uint8_cpu> 
Jan 13 21:58:27 device = 'CPU' 
Jan 13 21:58:27  
Jan 13 21:58:27     def run(test_self, device):  # type: (Any, Text) -> None 
Jan 13 21:58:27         if model_test.model_dir is None: 
Jan 13 21:58:27             model_dir = self.prepare_model_data(model_test) 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

houseroad committed:
Automatic update of fbcode/onnx to 65020daafa9183c769938b4512ce543fd5740f8f (#32125)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/32125

Previous import was 57ebc587fcf3913b4be93653b0dd58c686447298

Included changes:
- **[65020daa](https://github.com/onnx/onnx/commit/65020daa)**: better error message for undefined inputs (#2540) <Yuxin Wu>
- **[8afff0e9](https://github.com/onnx/onnx/commit/8afff0e9)**: bump ORT version (#2538) <Lu Fang>
- **[3d9ca57e](https://github.com/onnx/onnx/commit/3d9ca57e)**: fix name of directory (#2537) <Prasanth Pulavarthi>
- **[df8fa2c9](https://github.com/onnx/onnx/commit/df8fa2c9)**: Repository guidelines (#2539) <Prasanth Pulavarthi>
- **[49cc2f02](https://github.com/onnx/onnx/commit/49cc2f02)**: Update CircleCI job to use Python3.6 (#2527) <bddppq>
- **[25ff79a4](https://github.com/onnx/onnx/commit/25ff79a4)**: Fix wrong model version, it's not 12 (the onnx_opset_version()), not 11 (the opset version of the latest stable), but 10 (#2478) <daquexian>
- **[7cebaed5](https://github.com/onnx/onnx/commit/7cebaed5)**: Fix Windows py3.5 CI (#2529) <bddppq>
- **[eddae00e](https://github.com/onnx/onnx/commit/eddae00e)**: Correct the order of arguments of InferShapes (#2500) <Shinichiro Hamaji>
- **[41b5afe6](https://github.com/onnx/onnx/commit/41b5afe6)**: Include <ostream> in common/status.h (#2519) <Casey Carter>
- **[423f1977](https://github.com/onnx/onnx/commit/423f1977)**: add 8 bit support to maxpool op (#2510) <Ashwini Khade>
- **[78593c2f](https://github.com/onnx/onnx/commit/78593c2f)**: add 8 bit support to reducemin and reducemax ops (#2516) <Ashwini Khade>

Test Plan: cont build

Differential Revision: D19380034

fbshipit-source-id: 1ea677eed6779d2b3f8e4683225ba856c68159cd
